### PR TITLE
Move more files out of platform, add folders

### PR DESCRIPTION
**Pull request by sanlouise**
## Description
This PR is the fifth in a series to move and rename logic that's reused from profile 1. I divided up the larger task in the ticket into smaller PRs as to not touch too many files at once and not overwhelm reviewers.

- [First PR](https://github.com/department-of-veterans-affairs/vets-website/pull/14691)
- [Second PR](https://github.com/department-of-veterans-affairs/vets-website/pull/14698)
- [Third PR](https://github.com/department-of-veterans-affairs/vets-website/pull/14710)
- [Fourth PR](https://github.com/department-of-veterans-affairs/vets-website/pull/14729)

This PR
- Moves components that are exclusively used by the profile out of `platform` and into the profile application
- Adds specific sub-folders to `src/applications/personalization/profile/components` to organize components based on each profile page
- Adds specific sub-folders to `src/applications/personalization/profile/components/personal-information` to organize components on the Personal Information page

## Testing done
All looks good locally.

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by rianfowler — APPROVED** [review]

File: src/applications/personalization/profile/tests/components/direct-deposit/PaymentInformationBlocked.unit.spec.jsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { shallow } from 'enzyme';
 import { expect } from 'chai';
 
-import PaymentInformationBlocked from '../../components/PaymentInformationBlocked';
+import PaymentInformationBlocked from 'applications/personalization/profile/components/direct-deposit/PaymentInformationBlocked';
```
Comment (erikphansen) [inline src/applications/personalization/profile/tests/components/direct-deposit/PaymentInformationBlocked.unit.spec.jsx:5]: We _could_ set up a new Babel alias ([like we have for vet360](https://github.com/department-of-veterans-affairs/vets-website/blob/369b26b47357b6f049a1fd3abe59b9aa49a76b6f/.babelrc#L54)) so we could just do `import PaymentInformationBlocked from 'profile/components/direct-deposit/PaymentInformationBlocked'`. I'm a fan of that, but also understand the argument that that can lead to confusion.

File: src/applications/personalization/profile/tests/components/alerts/Profile.data-unavailable-error.unit.spec.js [hunk 2]
```diff
@@ -7,11 +7,11 @@ import { setupServer } from 'msw/node';
 
 import { resetFetch } from 'platform/testing/unit/helpers';
 
-import * as mocks from '../../msw-mocks';
-import { renderWithProfileReducers as render } from '../unit-test-helpers';
+import * as mocks from '../../../msw-mocks';
```
Comment (erikphansen) [inline src/applications/personalization/profile/tests/components/alerts/Profile.data-unavailable-error.unit.spec.js:10]: ...and if we added that Babel alias this import would become `import * as mocks from 'profile/msw-mocks'`;

**Review by erikphansen — APPROVED** [review]
I think we should set up a Babel alias for `applications/personalization/profile` since that import is being used all over the place now. If we used something like `#profile` for the alias (instead of just `profile`) I think that would help prevent some of the confusion ("is this importing something from `node_modules`?") that can arise from using aliases.

[This blog post](https://medium.com/bootstart/why-you-should-use-babel-resolvers-210615fc41d) goes into more detail about avoiding this sort of confusion. It also makes a point that I like: for our imports that look like `... from 'applications/...` we should be prefixing them with a slash `... from '/applications/..` and I really wish I'd done that when I initially set up Bable aliases :(

File: src/applications/personalization/profile/tests/components/direct-deposit/PaymentInformationBlocked.unit.spec.jsx (hunk 1 above)
Comment (sanlouise) [reply src/applications/personalization/profile/tests/components/direct-deposit/PaymentInformationBlocked.unit.spec.jsx:5]: I like this approach a lot! Let me explore this in my follow up PR. Thanks for pointing this out!

**Pull request merged**